modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,68 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2 Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are complete, the backup is marked as complete.
+
+
+[NOTE]
+====
+The OADP 1.1 Data Mover is a Technology Preview feature.
+
+The OADP 1.2 Data Mover has significantly improved features and performances, but is still a Technology Preview feature.
+====
+:FeatureName: The OADP Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends that customers who use OADP 1.2 Data Mover in order to back up and restore ODF CephFS volumes, upgrade or install OCP 4.12+ for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing, can improve the performance of backup times.
+
+* https://issues.redhat.com/browse/RHSTOR-4287[CephFS ROX details]
+//* https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md[Provisioning and mounting CephFS snapshot-backed volumes]
+
+
+//For more information about OADP 1.2 with CephS [name of topic], see ___.
+
+====
 
 .Prerequisites
 
 * You have verified that the `StorageClass` and `VolumeSnapshotClass` custom resources (CRs) support CSI.
 
 * You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
++
+[NOTE]
+====
+In OCP 4.12+ verify that this is the only default `volumeSnapshotClass`.
```

Comment:
"OCP" jumped out at me while I scanned for merge review. It's not an official product name, replace with `{product-title}`. Otherwise I think this is ready to merge :+1: 

```suggestion
In {product-title} version 4.12 or later, verify that this is the only default `volumeSnapshotClass`.
```

Reply:
@jeana-redhat 
Thank you. I implemented those changes.
Please merge the PR.
Thank you for your help!
Carmi